tactical-microgrid-standard/cli/CLIClient.h

```diff
@@ -58,7 +59,9 @@ class CLIClient : public TimerHandler<UnavailableController> {
   // since its last heartbeat, the controller is deemed unavailable.
   // The delay is the sum of the missed controller delay (3s) and the lost controller
   // delay (6s) from the TMS spec.
-  static constexpr Sec unavail_controller_delay = Sec(3 + 6);
+  static constexpr Sec missed_controller_delay = Sec(3);
```

Comment:
```suggestion
  static constexpr Sec missed_controller_delay {3};
```
This doesn't need to repeat the type in the initializer. It can use any form of init (if it compiles)
` = 3`
`(3)`
` = {3}`
`{3}`